source/main/ipc-api/get-logs.js

```diff
@@ -40,13 +23,16 @@ export default () => {
     const logFiles = [];
     if (fs.existsSync(pubLogsFolderPath)) {
       const files = fs.readdirSync(pubLogsFolderPath);
```

Comment:
can you split your loop into two parts: one for Daedalus files and one for node files?
and then, read the node log directory using the regexp `ALLOWED_NODE_LOGS`?
the reason I ask is because we want to add a symlink to the recent log file, and this symlink should not be included in the log file list.

Reply:
also, I don't know if `fs.readdirSync` will return the file list sorted. Maybe sorting it is a more safe option.

Reply:
@CodiePP not sure I understand the need for 2 loops since both will loop through the same files list. What is the symlink file name?